### DISCUSSION TITLE
如何在Waline中显示之前Valine导入的评论？

### DISCUSSION THREAD

**Discussion by LxnChan** [walinejs/waline#1158] [Q&A] [answered]
之前的时候用的waline旧版本能直接显示valine的评论，现在好像显示不出来了，看起来应该是差点东西，不知道有没有脚本能转换或者添加上缺失项的。

补充：部署方式：腾讯云开发tcb

之前的：
![image](https://user-images.githubusercontent.com/46388610/176202961-c36ec123-bedd-44cf-b325-aaea8c3ed4d9.png)

现在的：
![image](https://user-images.githubusercontent.com/46388610/176203298-75b862d5-0602-4666-94cb-bdf62933e1ee.png)

**Comment by lizheming** [comment, marked as answer]
需要提供下相关的数据和有问题的地址看一下，理论上不存在新旧的差别的

**Reply by LxnChan** [reply]
相关数据上面那个图就是，两张图中上面那个图是显示不出来的，下面那个图是能显示出来的。

有问题的地址：我站很多页面都有这个情况，https://lxnchan.cn/about/ 是其中一个。

另外旧版本的评论能够在管理面板中显示。

**Reply by LxnChan** [reply]
> 需要提供下相关的数据和有问题的地址看一下，理论上不存在新旧的差别的

应该是时间戳的问题，新版本Waline存储和读取的是date类型的时间戳，旧版本Valine读取的是string类型的时间。

**Reply by lizheming** [reply]
@LxnChan 不太清楚你这个老数据是如何导入到 CloudBase 的了，{__type, iso} 这种格式是 LeanCloud 的格式，不是 CloudBase 支持的格式。要处理你这个问题，可以把评论数据导出出来，然后把 {__type, iso} 批量修改成 {"$date":""} 这个格式应该就可以了。
